joomla/joomla-cms
# [4.0] Cleanup VoteRadio field

### Summary of Changes

Cleans up `Joomla\Component\Content\Administrator\Field\VoteRadioField` so that it's either attached to the form or not instead of hiding/showing label and options.

### Testing Instructions

Go to Articles configuration.
In `List Layouts` tab check that `Show Votes in List` and `Show Ratings in List` fields appear when `Content - Vote` plugin is enabled. And don't appear when the plugin is disabled.

### Documentation Changes Required

No.
